texttospeech/api/TextToSpeech/TextToSpeech.cs

```diff
@@ -0,0 +1,288 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using CommandLine;
+using System;
+using System.IO;
+using Google.Cloud.TextToSpeech.V1;
+
+namespace GoogleCloudSamples
+{
+
+    [Verb("list", HelpText = "List available voices.")]
+    class ListArgs
+    {}
+
+    [Verb("synthesize", HelpText = "Synthesize input to audio")]
+    class SynthesizeArgs
+    {
+        [Value(0, HelpText = "The text to translate.",
+            Required = true)]
+        public string Text { get; set; }
+
+        [Option('f', HelpText = "Source formatting")]
+        public string SourceFormat { get; set; }
+
+    }
+
+    [Verb("synthesize-file", HelpText = "Synthesize a file to audio")]
+    class SynthesizeFileArgs
+    {
+        [Value(0, HelpText = "The file to synthesize",
+            Required = true)]
+        public string Text { get; set; }
+
+        [Option('f', HelpText = "Source formatting")]
+        public string SourceFormat { get; set; }
+    }
+
+    public class TextToSpeech
+    {
+        public static void Main(string[] args)
+        {
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+            Parser.Default.ParseArguments<ListArgs, SynthesizeArgs,
+                SynthesizeFileArgs>(args).MapResult(
+                (ListArgs largs) => ListVoices(largs),
+                (SynthesizeArgs sargs) => Synthesize(sargs),
+                (SynthesizeFileArgs sfargs) => SynthesizeFile(sfargs),
+                errs => 1);
+        }
+
+        private static int ListVoices(ListArgs args)
+        {
+            ListVoices();
+            return 0;
+        }
+
+        // [START tts_list_voices]
+        /// <summary>
+        /// Lists all the voices available for speech synthesis.
+        /// </summary>
+        public static void ListVoices()
+        {
+            TextToSpeechClient client = TextToSpeechClient.Create();
+
+            // Performs the list voices request
+            var response = client.ListVoices(new ListVoicesRequest
+            {
+                // Uncomment the following block and specify a language
+                // code to get results for that language only.
+                //LanguageCode = "[LANGUAGE_CODE]"
+            });
+
+            foreach (Voice voice in response.Voices)
+            {
+                // Display the voices's name.
+                Console.WriteLine($"Name: {voice.Name}");
+
+                // Display the supported language codes for this voice.
+                foreach (var languageCode in voice.LanguageCodes)
+                {
+                    Console.WriteLine($"Supported language(s): {languageCode}");
+                }
+
+                // Display the SSML Voice Gender
+                Console.WriteLine("SSML Voice Gender: " +
+                    (SsmlVoiceGender)voice.SsmlGender);
```

Comment:
How about doing this without the casting

`Console.WriteLine($"SSML Voice Gender: ${voice.SsmlGender}")`
or
`Console.WriteLine("SSML Voice Gender: {0}", voice.SsmlGender)`

Reply:
Ack.

voice.SsmlGender is a number.